perception/autoware_occupancy_grid_map_outlier_filter/src/occupancy_grid_map_outlier_filter_node.cpp

```diff
@@ -263,11 +265,24 @@ OccupancyGridMapOutlierFilterComponent::OccupancyGridMapOutlierFilterComponent(
   if (enable_debugger) {
     debugger_ptr_ = std::make_shared<Debugger>(*this);
   }
+
+  // time keeper
+  bool use_time_keeper = declare_parameter<bool>("publish_processing_time_detail");
```

Comment:
@YoshiRi 
The package 'autoware_occupancy_grid_map_outlier_filter' do not have separate parameter file (usually in `<package>/config/*.param.yaml`). Do we need to add a parameter file only for the debug config?

If there is not config file, the default value is outside of the package. Then, shall we add a default value on the declaration?

Reply:
Here is a PR that make parameter files.
https://github.com/autowarefoundation/autoware.universe/pull/6745

It would be better to align with this PR.